test/e2e/curated_packages_test.go

```diff
@@ -4,32 +4,173 @@
 package e2e
 
 import (
+	"context"
 	"fmt"
+	"strconv"
 	"strings"
 	"testing"
+	"time"
 
 	"github.com/aws/eks-anywhere/internal/pkg/api"
 	"github.com/aws/eks-anywhere/pkg/api/v1alpha1"
 	"github.com/aws/eks-anywhere/test/framework"
 )
 
 const (
-	eksaPackageControllerHelmChartName = "eks-anywhere-packages"
-	eksaPackageControllerHelmURI       = "oci://public.ecr.aws/eks-anywhere/eks-anywhere-packages"
-	eksaPackageControllerHelmVersion   = "0.1.10-eks-a-10"
-	eksaPackageBundleBaseURI           = "oci://public.ecr.aws/eks-anywhere/eks-anywhere-packages-bundles"
+	EksaPackagesNamespace              = "eksa-packages"
```

Comment:
These are test-specific values and should not be exported. The could probably be renamed and make it obvious that they're meant for the hello-eks-anywhere test, but these were not exported for a reason---they don't apply to other tests!

At the time I wrote these tests, there were no other artifacts against which I could test, so these were hard-coded in. It wasn't the best decision, but it was the only choice I had at the time.

Borrowing these values is not the right way forward, unless it's also the only way forward for metallb as well? And even still, I wouldn't recommend exporting these and using them _anywhere_ else.

Reply:
We had a voice chat, end result, we're leaving these here. I'd prefer if they were at least commented with a warning that they are here for convenience and any given test should use the values that make sense for them, not necessarily these values. But even that is hopefully moot since a good chunk of this will eventually change as we further develop our e2e test infrastructure.